source/sass/blocks/form.scss

```
@@ -10,17 +10,22 @@
 .form__fieldset {
   box-sizing: border-box;
   margin: 0 auto;
-  width: 320px;
-  padding: 0 20px;
+  width: 100%;
+  padding: 0 60px;
```

А почему собственно паддинги по краям 60px, а не 20? До этого было всё правильно. Группы полей в наешм варианте выполняют ещё и роль центровщика, поэтому вместо `width: 100%` должно быть `width: 320px` (как было у вас до этого). Итого, должно быть так:
```suggestion
  width: 320px;
  padding: 0 20px;
```